foreshadow/tests/test_cleaners/test_data_cleaner.py

```diff
@@ -126,11 +127,26 @@ def test_drop():
     columns = ["financials"]
     data = pd.DataFrame({"financials": ["", "", "", ""]}, columns=columns)
     cs = ColumnSharer()
-    dc = DataCleaner(cs)
+    dc = DataCleaner(column_sharer=cs)
     dc.fit(data)
     transformed_data = dc.transform(data)
     assert transformed_data.empty
     assert transformed_data.columns == columns
 
 
+def test_numerical_input():
+    """Test numerical input."""
+    import numpy as np
+    import pandas as pd
+    from foreshadow.cleaners import DataCleaner
+    from foreshadow.core.column_sharer import ColumnSharer
+
+    columns = ["financials"]
+    data = pd.DataFrame({"financials": np.arange(10)}, columns=columns)
+    cs = ColumnSharer()
+    dc = DataCleaner(column_sharer=cs)
+    dc.fit(data)
+    # transformed_data = dc.transform(data)
```

Comment:
I might make this a fit_transform so both are tested.

Reply:
I'll make a separate test with fit_transform (since the internal call structure for each is different and we should have coverage on each separately).